packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -45,7 +45,7 @@ function UncontrolledInnerBlocks( props ) {
 		forwardedRef,
 		templateInsertUpdatesSelection,
 		__experimentalCaptureToolbars: captureToolbars,
-		__experimentalMoverDirection,
+		__experimentalBlockListOrientation,
```

Comment:
maybe it could be just "orientation" and make it stable? do we expect any other changes to this API?

Reply:
I don't think so, sounds good to me to stabilize it.